app/models/devise_token_auth/concerns/tokens_serialization.rb

```diff
@@ -0,0 +1,19 @@
+module DeviseTokenAuth::TokensSerialization
+  # Serialization hash to json
+  def self.dump(object)
+    object.each_value(&:compact!) unless object.nil?
```

Comment:
If we care about storing `null` values we can delete this line.
See `last_token` below:

"{\"nlw47IfMSJxHl8v2bvhrOw\":{\"token\":\"$2a$10$3yWVUhPtnn/vigTztah2fuioMMEB6R8aZunkFOjzguV1ZCZ8kga16\",\"expiry\":1549026407,\"last_token\":null,\"updated_at\":\"2019-01-18T13:06:32.876Z\"}}"